helper/ttl/validation.go

```diff
@@ -0,0 +1,83 @@
+package ttl
+
+import (
+	"errors"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/vault/logical"
+)
+
+type MountHandler struct {
```

Comment:
I don't see the connection between "MountHandler" and this TTL configuration. For the struct, something like "TTLConfig" is a good descriptor, but is awkward with "ConfigTTL" (which is fine) in the struct. TTLSettings? Still thinking on this one...

Reply:
Yeah, I'm not sure the name is perfect. Since the package's name is `ttl`, outside the package it will be referred to as a `ttl.MountHandler`, and someday may have more than just a Validate method? So keep that in mind for naming. Definitely can't use the name TTL in the struct name because that goes straight into stutter land.

Reply:
`ttl.Settings` or `ttl.MountSettings`?. "Handler" often denotes processing requests which is probably my main concern with the name.

Reply:
Yeah I do like that better. Handler is such a lame catch-all. I'll update that. :-) I should omit it from my vernacular except for actual http request handlers. I need to for get the word "Manager" too.